### PR TITLE
Hsh feature release 0905

### PR DESCRIPTION
**Pull request by fergie-nz**
Adds docs for new release features for HSH:

Adds docs for:

- [x] Governance work flows
- [x] Logging displays:
    - [x] Supplier users
    - [x] Supplier tender
    - [x] Customer users
    - [x] Customer tender
- [x] New supplier registration process
- [x] Audit log report
- [x] Organisation profiles for suppliers

### CODE REVIEW

**Review by Chris-Petty — CHANGES_REQUESTED** [review]
Really just concerned with whether or not we make all the contact support references consistent referring to Health Supply Hub Support Team rather than mSupply Support Team?

File: content/health-supply-hub/supplier/submitting_bids.md
[the remark this answers is not in the document]
Comment (Chris-Petty) [reply content/health-supply-hub/supplier/submitting_bids.md:1]: TBF I think the latter looks nicer if if the grouping doesn't necessarily make sense?

Old
image
New
image

File: content/health-supply-hub/supplier/submitting_bids.md [hunk 1]
```diff
@@ -72,74 +72,80 @@ It is your sole responsibility to engage with the relevant Purchaser to indicate
...
 6. In addition to submitting your product bids you must also ensure that you have provided the Purchaser with all the additional requested documentation. You can upload these documents to HSH by clicking the Attachment button and dragging and dropping the files in the greyed out section or click the greyed out section to search for them.
 
-![Supplier upload attachments](/health-supply-hub/images/supplier_attach_files.png)
+![Supplier upload attachments](/health-supply-hub/supplier/images/supplier_attach_files.png)
 
 > IMPORTANT: If your quote has been uploaded, but not submitted, the Purchaser will not be able to see your quote. Ensure your quote status is displayed as "Quote Submitted" before the closing date.
 
 ## Video Tutorial - Add attachments to your quote
 
 {{ youtube(id="Mky2cJ_hNUU", class="youtube") }}
```
[the remark this answers is not in the document]
Comment (fergie-nz) [reply content/health-supply-hub/supplier/submitting_bids.md:140]: oh curious! I hadn't ever actually played it

File: content/health-supply-hub/supplier/submitting_bids.md
[the remark this answers is not in the document]
Comment (fergie-nz) [reply content/health-supply-hub/supplier/submitting_bids.md:1]: > Not sure if I agree with the weight changes to the headings but not fussed either heh

I was roughly trying to standardise - but no strong opinions on anything. Do you think reverting better?

File: content/health-supply-hub/supplier/managing_manufacturers.md [hunk 2]
```diff
@@ -16,23 +16,23 @@ top = false
 
 Identifying the correct final dose form manufacturer is an important step to ensure customers understand the product being supplied.
 
-The Health Supply Hub maintains list of manufacturers to select from. If the manufacturer you require is not on the list, please fill out the manufacturer request form [here](https://docs.google.com/forms/d/e/1FAIpQLScv8Mmin1SXuJwD7MDALEYb648Nrb_XvLpBUer8YIhdwR9NpA/viewform?usp=sf_link), or if you have a number of manufacturers to add, please email `health-supply-hub@msupply.foundation` attaching the list of manufacturers to add.
+The Health Supply Hub maintains list of manufacturers to select from. If the manufacturer you require is not on the list, please fill out the manufacturer request form [here](https://docs.google.com/forms/d/e/1FAIpQLScv8Mmin1SXuJwD7MDALEYb648Nrb_XvLpBUer8YIhdwR9NpA/viewform?usp=sf_link), or if you have a number of manufacturers to add, please email [Health Supply Hub Support Team](mailto:health-supply-hub@msupply.foundation) and attach the list of manufacturers to add.
```
[the remark this answers is not in the document]
Comment (fergie-nz) [reply content/health-supply-hub/supplier/managing_manufacturers.md:19]: Yeah agreed. I might ask now how it is being handled currently

File: content/health-supply-hub/supplier/submitting_bids.md
[the remark this answers is not in the document]
Comment (Chris-Petty) [reply content/health-supply-hub/supplier/submitting_bids.md:1]: nah

**Pull request merged**